pages/common/cargo-owner.md

```diff
@@ -0,0 +1,20 @@
+# cargo owner
+
+> Manage the owners of a crate on the registry.
+> More information: <https://doc.rust-lang.org/cargo/commands/cargo-owner.html>.
+
+- Invite the given user or team as an owner:
+
+`cargo owner --add {{login}} {{crate}}`
+
+- Remove the given user or team as an owner:
+
+`cargo owner --remove {{login}} {{crate}}`
```

Comment:
```suggestion
`cargo owner --remove {{name}} {{crate}}`
```

Or maybe `user_name`. This way it is clear that this is an identifier for a user or group.

Reply:
I'll add a more descriptive placeholder from https://doc.rust-lang.org/cargo/reference/publishing.html#cargo-owner